src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Tagging {
+	public enum TagAction {ADD, REMOVE};
```

Comment:
Great! By using enum, you can extend the functionality easily if we want to support another action such as `Edit`